css/pages/_team.scss

```diff
@@ -0,0 +1,133 @@
+.main-content-title{
+  padding-bottom: 20px;
+  color: transparent;
+  background-image: $gradient-red-text-color;
+  -webkit-background-clip: text;
+  background-clip: text;
+  @include hero-title;
+  @include media-breakpoint-up(md) {
+    padding-left: 20px;
+  }
+  @include media-breakpoint-up(lg) {
+    padding: 20px 120px 40px;
+  }
+  @include media-breakpoint-up(xl) {
+    padding-top: 60px;
+    padding-bottom: 80px;
+  }
+  @include media-breakpoint-up(xxl) {
+    padding-right: 240px;
+    padding-left: 240px;
+  }
+}
+.layout-grid {
+  @include media-breakpoint-up(md) {
+    display: grid;
+    column-gap: 10px;
+    grid-template-columns: 1fr 1fr;
+    padding-bottom: 20px;
+  }
+  @include media-breakpoint-up(lg) {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    padding-bottom: 40px;
+    gap: 40px;
+  }
+  @include media-breakpoint-up(xl) {
+    padding-bottom: 80px;
+  }
+}
+.main-content-link {
+  display: inline-flex;
+  margin-bottom: 20px;
+  @include media-breakpoint-up(lg) {
+    flex-direction: column;
+  }
+}
+.grid-item-first {
+  @include media-breakpoint-up(lg) {
+    grid-column: 1 / 2;
+    grid-row: 1 / 2;
+  }
+}
+.grid-item-second {
+  @include media-breakpoint-up(lg) {
+    grid-column: 2 / 3;
+    grid-row: 1 / 2;
+  }
+}
+.grid-item-third {
+  @include media-breakpoint-up(lg) {
+    grid-column: 3 / 4;
+    grid-row: 1 / 2;
+  }
+}
+.grid-item-fourth {
+  @include media-breakpoint-up(lg) {
+    grid-column: 3 / 4;
+    grid-row: 2 / 3;
+  }
+}
+.grid-item-fifth {
+  @include media-breakpoint-up(lg) {
+    grid-column: 2 / 3;
+    grid-row: 2 / 3;
+  }
+}
+.grid-item-sixth {
+  @include media-breakpoint-up(lg) {
+    grid-column: 1 / 2;
+    grid-row: 2 / 3;
```

Comment:
Why do you need this styles? I am sure you can create such a simple grid-layout without these properties.
And also you can use bootstrap for this section instead of grid.

Reply:
I add these properties because the order of second-row cards is changing. I thought this is a good way. 
I need three of them (start with line 65 end - line 80), or create the layout differently?